### PR TITLE
Introduce Dockerfile and GitHub Action support

### PR DESCRIPTION
**Pull request by michaelmelody91**
Addition of a Dockerfile to enable docker support. Addition of GitHub action which utilises the Dockerfile so that CLI commands can be made on GitHub events.

### Example Action: Update API

```yaml
name: Push API
on:
  push:
    branches:
      - develop
      
env:
  SWAGGERHUB_API_KEY: ${{ secrets.SWAGGERHUB_API_KEY }}
  SWAGGERHUB_URL: ${{ secrets.SWAGGERHUB_URL }}
jobs:
  build:
    runs-on: ubuntu-latest
    name: Update SwaggerHub
    steps:
    - name: Checkout
      uses: actions/checkout@v2
    - name: Call CLI
      uses: smartbear/swaggerhub-cli@pr_demo
      with:
        args: api:update MyOrg/SimpleApi -f ./docs/api.yml
```

### CODE REVIEW

File: .github/workflows/swg-cli.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+    name: A job to verify action
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Call CLI
+      id: swg-cli
+      uses: michaelmelody91/swaggerhub-cli@pr_demo
+      with:
+        args: api:get michaelmelody/SimpleApi
```
Comment (michaelmelody91) [inline .github/workflows/swg-cli.yaml:20]: Not to be included. Will remove after review.

**Review by conor-ward — APPROVED** [review]
Looks good

**Review by barryadk — DISMISSED** [review]

File: .github/workflows/swg-cli.yaml (hunk 1 above)
Comment (barryadk) [reply .github/workflows/swg-cli.yaml:20]: Cheers @michaelmelody91 I can merge once you update the PR.

Comment (michaelmelody91) [reply .github/workflows/swg-cli.yaml]: Done, thanks @barryadk!

**Review by barryadk — APPROVED** [review]

**Comment by michaelmelody91** [comment]
🥳

**Pull request merged**